pom.xml

```diff
@@ -1897,6 +1897,11 @@
                 <artifactId>metrics-jvm</artifactId>
                 <version>${metrics.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.dropwizard.metrics</groupId>
+                <artifactId>metrics-jmx</artifactId>
+                <version>${metrics.version}</version>
```

Comment:
when we use properties only once, It's better to inline them. Could you benefit from this PR to do the inlining ?

Reply:
I see `${metrics.version}` is used in other dependencies `metrics-jvm`, `metrics-core`